File: front/src/app/models/pontoNormalizado.ts
export class PontoNormalizado {
    entrada: Date;
    saida: Date;
    jornadaString: string;
    tarefas: string;

    constructor(item?) {
        if (item) {
            this.entrada = new Date(item.entrada);
            this.saida = new Date(item.saida);
            this.jornadaString = item.jornadaString;
            this.tarefas = item.tarefas;
        }
    }
}

File: back/Infra/Views/PontoNormalizado.cs
using System;
using System.Diagnostics.CodeAnalysis;

namespace SGE.Infra.Views
{
    [ExcludeFromCodeCoverage]
    public class PontoNormalizado
    {
        public DateTimeOffset Entrada { get; set; }

        public DateTimeOffset Saida { get; set; }

        public DateTimeOffset Jornada { get; set; }

        public string JornadaString { get; set; }

        public string Tarefas { get; set; }
    }
}